desktopexporter/internal/server/server.go

```diff
@@ -1,16 +1,16 @@
 package server
 
 import (
+	"context"
 	"embed"
 	"encoding/json"
-	"fmt"
 	"io/fs"
 	"log"
 	"net/http"
 	"os"
+	"strconv"
 	"time"
 
-	"github.com/gorilla/mux"
```

Comment:
Unless I'm messing something up locally, this directory needs a `go mod tidy` since we dropped a dependency